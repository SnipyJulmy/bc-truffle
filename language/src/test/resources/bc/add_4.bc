1 + 2
2 + 1
0 + 0
10 + 0
0 + 10
4 + 11
3129832913 + 13213214143
128371237 + 128732138721391 + 321837213 + 93210873921 + 2139021 + 1232142
128371237 + 128732138721391 + 321837213 + 93210873921 + 2139021 + 1232142 + 123 + 987654
4.5 + 3.2
halt
