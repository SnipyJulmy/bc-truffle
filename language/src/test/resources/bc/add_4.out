3
3
0
10
10
15
16343047056
128825803174925
128825804162702
7.7
